repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/opentracing/httpclient/ReposeTracingRequestInterceptor.scala

```diff
@@ -81,8 +81,6 @@ class ReposeTracingRequestInterceptor(tracer: Tracer, reposeVersion: String, uri
 
       httpContext.setAttribute(OpenTracingSpan, clientSpan)
 
-      if (Option(tracer.scopeManager.active).isEmpty) logger.warn("Current scope is null; possibly failed to start client tracing span.")
```

Comment:
This log message would indicate that there was no active span when we create a new span for outbound requests.

Unfortunately, that is always the case when the OpenTracing Service is not configured for use.

Really, a failure to create/activate a span should be logged wherever that span is created/activated, not when we are making this new span. After all, what if deactivating the active span was intentional? Say the `ReposeFilter` started a new span for the request, but a filter in the filter chain intentionally deactivated that span. My point is that this method does not have enough data to determine whether or not there should be an active span. Plus, you could always check the OpenTracing collection service to figure out whether or not a span has a parent.